modules/ROOT/pages/configure-openshift-routes.adoc

```diff
@@ -0,0 +1,99 @@
+= Configuring Red Hat OpenShift Routes in Runtime Fabric
+
+Runtime Fabric enables you to configure Red Hat OpenShift Routes to create ingress endpoints for your Mule applications. 
+
+== Before You Begin
+
+Configuration of routes in Runtime Fabric follows closely to ingress resource configuration. Before you create routes, review the following:
+* xref:custom-ingress-configuration.adoc#how-ingress-resources-templates-work-in-runtime-fabric[How Ingress Resources Templates Work in Runtime Fabric] 
+* xref:custom-ingress-configuration.adoc#example-ingress-resource-templates[Example Ingress Resource Templates]
+
+This documentation also assumes you’re familiar with https://docs.openshift.com/container-platform/4.10/networking/routes/route-configuration.html#nw-ingress-creating-a-route-via-an-ingress_route-configuration[Creating a route through an Ingress object^].
+
+== Example Ingress Resource Template for A Red Hat OpenShift Route
+The following example illustrates how to modify a Kubernetes ingress spec to create a Runtime Fabric ingress resource template for the networking.k8s.io/v1 API. 
+
+[source,yaml]
+--
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: example-rtf-ingress
+  namespace: rtf (1)
+  labels:
+spec:
+  ingressClassName: rtf-openshift (2)
+  Tls: (3)
+  - hosts:
+      - app-name.example.com
+    secretName: example-tls
+  rules:
+    - host: app-name.example.com (4)
+      http:
+        Paths: (5)
+        - path: / (6)
+          pathType:
+          backend: (7)
+            serviceName: service-name
+            servicePort: 80
+--
+
+Note the following about this example:
+
+<1> The template must be placed in the `rtf` namespace.
+
+<2> `ingressClassName` must be prefixed with `rtf`-, and must be set to `rtf-openshift`.
```

Comment:
```suggestion
<2> `ingressClassName` must be prefixed with `rtf-`, and must be set to `rtf-openshift`.
```
This line is a bit confusing, if it is always set to `rtf-openshift` why do we mention the `rtf-` prefix? If the **openshift** part can change, do we explain in which scenarios? Otherwise, just say: 
`ingressClassName` must be set to `rtf-openshift`.

Reply:
No, in this case it has to be rtf-openshift